src/components/MonthlyCalendar/readme.md

```diff
@@ -199,3 +199,379 @@ const initialState = {
     </Card>
 </div>
 ```
+
+##### MonthlyCalendar interacting with events
+
+```js
+import React from 'react';
+import { Card, MonthlyCalendar, RenderIf, Drawer, Avatar, Button, Badge } from 'react-rainbow-components';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarAlt, faCheckCircle } from '@fortawesome/free-regular-svg-icons';
+
+const StyledContainer = styled.div
+`
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    align-items: start;
+    justify-content: flex-end;
+    padding: 4px;
+`;
+
+const StyledAssignedLabel = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    text-transform: uppercase;
+    font-size: 12px;
+    cursor: pointer;
+    padding-left: 2px;
+    color: ${props => props.text.header};
+`;
+
+const StyledAvailableLabel = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    text-transform: uppercase;
+    cursor: pointer;
+    font-size: 12px;
+    border-radius: 1rem;
+    margin: 4px 0 2px;
+    padding: 2px 10px 3px 9px;
+    color: ${props => props.getContrastText(props.text.main)};
+    ${props => props.count <= 1 &&
+        `
+            background: ${props.error.main};
+        `}
+    ${props => props.count > 1 && props.count <= 3 &&
+        `
+            background: ${props.warning.main};
+        `}
+    ${props => props.count > 3 &&
+        `
+            background: ${props.brand.main};
+        `}
+`;
+
+function DailyTasks(props) {
+    const {
+        availableTasksCount, assignedTasksCount
+    } = props;
+
+    if (!availableTasksCount && !assignedTasksCount) return null;
+
+    return (
+        <StyledContainer>
+            {assignedTasksCount > 0 &&
+                <StyledAssignedLabel>
+                    {`${assignedTasksCount} Assigned`}
+                </StyledAssignedLabel>
+            }
+            {availableTasksCount > 0 &&
+                <StyledAvailableLabel count={availableTasksCount}>
+                    {`${availableTasksCount} Availables`}
+                </StyledAvailableLabel>
+            }
+        </StyledContainer>
+    );
+}
+
+function areDatesEqual(date1, date2) {
+    const first = new Date(date1);
+    const second = new Date(date2);
+
+    return first.getYear() === second.getYear()
+        && first.getMonth() === second.getMonth()
+        && first.getDate() === second.getDate();
+}
+
+function getAvailableTasksCountForDate(date, tasks) {
+    return tasks.filter(task => areDatesEqual(date, task.date) && !task.isAssigned).length;
```

Comment:
isAvailable no isAssigned

Reply:
the map does not contain the key isAvailable, so we know that it is available when isAssigned is false
example: `{ date: '2019-12-04', hour: '7:20 PM', isAssigned: false, description: 'Meeting with Jose Doe', isConfirmed: true }`